toolchain/check/check.cpp

```diff
@@ -61,16 +61,39 @@ static auto RenderImportKey(ImportKey import_key) -> std::string {
 static auto TrackImport(Map<ImportKey, UnitAndImports*>& api_map,
                         Map<ImportKey, Parse::NodeId>* explicit_import_map,
                         UnitAndImports& unit_info,
-                        Parse::Tree::PackagingNames import) -> void {
+                        Parse::Tree::PackagingNames import, bool fuzzing)
+    -> void {
   const auto& packaging = unit_info.parse_tree().packaging_decl();
 
   IdentifierId file_package_id =
       packaging ? packaging->names.package_id : IdentifierId::Invalid;
-  auto import_key = GetImportKey(unit_info, file_package_id, import);
+  const auto import_key = GetImportKey(unit_info, file_package_id, import);
+  const auto& [import_package_name, import_library_name] = import_key;
+
+  if (import_package_name == CppPackageName) {
+    if (import_library_name.empty()) {
+      // Clang is not crash-resilient.
+      CARBON_DIAGNOSTIC(CppInteropMissingLibrary, Error,
+                        "`{0}` import missing library", std::string);
+      unit_info.emitter.Emit(import.node_id, CppInteropMissingLibrary,
+                             CppPackageName.str());
+      return;
+    }
+    if (fuzzing) {
+      // Clang is not crash-resilient.
+      CARBON_DIAGNOSTIC(CppInteropFuzzing, Error,
+                        "`{0}` import found during fuzzing", std::string);
+      unit_info.emitter.Emit(import.node_id, CppInteropFuzzing,
+                             CppPackageName.str());
```

Comment:
```suggestion
                        "`{0}` import found during fuzzing", llvm::StringLiteral);
      unit_info.emitter.Emit(import.node_id, CppInteropFuzzing,
                             CppPackageName);
```

Reply:
`StringLiteral` seems to not be supported, but I may be missing something here.
https://github.com/carbon-language/carbon-lang/blob/9dc450e0af9920cc5bc77d1261294978c070ae7f/toolchain/diagnostics/diagnostic.h#L137

Reply:
Done.
FWIW, I wanted to avoid duplicating this string so we more easily modify it.